### ISSUE TITLE
Use a component instead of reopening

### ISSUE THREAD

**Issue by oskarrough** [Kuzirashi/ember-link-action#2]
Hi, would it be possible to extend `Ember.LinkComponent` with a new `LinkToAction` component instead of reopening? I made a quick failed attempt…

This would be to not pollute an already heavy LinkComponent.

**Comment by Kuzirashi** [comment]
I'll try.

**Comment by nathanhammond** [comment]
Better yet, just provide this as a mixin, the consumer would then be responsible for wiring it up.

**Comment by Kuzirashi** [comment]
I investigated this issue some time ago and I came up with a conclusion that I would have to create another addon to implement it as independent component. Or maybe there's some way to configure it via options passed to EmberApp?

I think it's easier to provide already working and everything setup addon than to give consumer mixin which he can implement. It would be faster then to use some workaround than to use my addon.
